ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/data/PurchaseInformation.kt

```diff
@@ -36,6 +36,11 @@ internal data class PurchaseInformation(
      * until the end of the billing period.
      */
     val isCancelled: Boolean,
+    /**
+     * Indicates whether this is a lifetime purchase.
+     * This is true for promotional lifetime purchases or non-subscription purchases attached to an entitlement.
```

Comment:
Shouldn't this be true as well for `non-subscription` that are not attached to an entitlement?